pinot-core/src/main/java/com/linkedin/pinot/core/startree/MetricBuffer.java

```diff
@@ -23,20 +23,17 @@
 
 
 public class MetricBuffer {
-  Number[] numbers;
+  private Number[] _numbers;
```

Comment:
Again, recommended to not change the naming convention within a file. I think the plan is to move to google coding standards which does not use leading '_'. Until that happens, we are refraining from explicitly modifying the names in a file (as long as it sticks to one convention).